### PR TITLE
Minor fixes from my lecture on 9/18/2024

### PR DESCRIPTION
**Pull request by sladenheim**
There were some issues with merging my code into the branch, so that is why there is a drawio update in here.

### CODE REVIEW

File: ds701_book/03-Probability-and-Statistics-Refresher.qmd [hunk 1]
```diff
@@ -296,7 +296,7 @@ Now if $B$ is a proper subset of $\Omega,$ then $P(B) < 1$.   So $P(\cdot|B)$ is
 :::
 ::: {.column width="40%"}
 
-![](drawio/ConditionalProbability.png){fig-align="center" width=80%}
+![](figs/Lec03-ConditionalProbability.png){fig-align="center" width=80%}
```
Comment (trgardos) [inline ds701_book/03-Probability-and-Statistics-Refresher.qmd:299]: would be nice to indicate somehow that these are drawio editable pngs. perhaps add '-drawio' to the end of the name?

File: ds701_book/05-Distances-Timeseries.qmd [hunk 2]
```diff
@@ -132,14 +132,18 @@ A metric is a function $d(x, y)$ that satisfies the following properties.
...
 ::: {.column width="30%"}
 :::: {.fragment}
-![](drawio/TriangleInequality.png){fig-align="center"}
+![](figs/Lec03-TriangleInequality.png){fig-align="center"}
 ::::
 :::
 ::::
 
 ::: {.fragment}
+<<<<<<< HEAD
```
Comment (trgardos) [inline ds701_book/05-Distances-Timeseries.qmd:141]: unresolved merge conflict

File: ds701_book/_freeze/03-Probability-and-Statistics-Refresher/figure-html/cell-15-output-1.png
Comment (trgardos) [inline ds701_book/_freeze/03-Probability-and-Statistics-Refresher/figure-html/cell-15-output-1.png:1]: i assume these were 'git mv'. does github only show moved files as deleted/added?

**Review by trgardos — CHANGES_REQUESTED** [review]
there are merge conflict markers in the qmd files

File: ds701_book/_freeze/03-Probability-and-Statistics-Refresher/figure-html/cell-15-output-1.png
Comment (sladenheim) [reply ds701_book/_freeze/03-Probability-and-Statistics-Refresher/figure-html/cell-15-output-1.png:1]: I think so. I have only been using git mv for changing file names.

File: ds701_book/03-Probability-and-Statistics-Refresher.qmd (hunk 1 above)
Comment (sladenheim) [reply ds701_book/03-Probability-and-Statistics-Refresher.qmd:299]: I'm reverting this back to the drawio folder. Then all drawio files are there and it is clear they are editable pngs.

**Review by trgardos — APPROVED** [review]

**Pull request merged**